examples/all-clusters-app/cc13x2x7_26x2x7/README.md

```diff
@@ -212,10 +211,10 @@ fully provisioned, BLE advertising will stop.
 #### Bluetooth LE Rendezvous
 
 Pairing this application with `ble-thread` can be done with any of the enabled
-[CHIP Controller](../../../src/controller/README.md) applications. Use the
-information printed on the console to aide in pairing the device. The controller
-application can also be used to control the example app with the cluster
-commands.
+[CHIP Controller](https://github.com/project-chip/connectedhomeip/blob/master/src/controller/README.md)
```

Comment:
Why can't we keep a relative link like in the case of example READMEs?

Reply:
Only files in the /docs and /examples folders are part of the documentation and can be linked to. I did however create a solution to make links to files in other parts of the repository into http-links to the github location. This should have been using that solution, thank you for pointing it out! It has now been fixed.